client/src/elm/Translate.elm

```diff
@@ -12968,6 +13048,78 @@ translationSet trans =
             , kinyarwanda = Nothing
             }
 
+        Send ->
+            { english = "Send"
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsApp ->
+            { english = "Send via WhatsApp"
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppComponentsSelectionHeader reportType ->
+            case reportType of
+                Components.SendViaWhatsAppDialog.Model.ReportWellChild ->
+                    { english = "Please select which sections of the Standard Pediatric Visit Report you would like to send:"
+                    , kinyarwanda = Nothing
+                    }
+
+                Components.SendViaWhatsAppDialog.Model.ReportAntenatal ->
+                    { english = "Please select which sections of the Antenatal Report you would like to send:"
+                    , kinyarwanda = Nothing
+                    }
+
+        SendViaWhatsAppConfirmationBeforeSendingHeader ->
+            { english = "By pressing send you are releasing the selected documents to:"
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppConfirmationBeforeSendingQuestion ->
+            { english = "Would you like to send?"
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppConsentQuestion ->
+            { english = "Does the patient consent to having their medical records sent via WhatsApp?"
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppNoticeOfNonRespobsibility ->
+            { english = "Please note that the medical professional and E-Heza will not be liable for what happens to these medical reports once released."
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppPhoneInputHeader ->
+            { english = "Enter the correct phone number for the patient:"
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppPhoneVerificationHeader ->
+            { english = "The phone number we have on file for this patient is:"
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppPhoneVerificationQuestion ->
+            { english = "Is this the correct number for the patients WhatsApp?"
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppPhoneUpdateAtProfileQuestionPrefix ->
+            { english = "Would you like to update the patient profile for"
+            , kinyarwanda = Nothing
+            }
+
+        SendViaWhatsAppPhoneUpdateAtProfileQuestionSuffix ->
+            { english = "with the number"
```

Comment:
If it would be PHP + Drupal, we'd complain that this is not really translatable. Not sure about Kinyarwanda language, but splitting a sentence into chucks typically cause translatability issues.

Reply:
@AronNovak The thing is that there requiremetn is to have patient name and phone number as part of the sentence, and they should have blue color:

![Selection_1462](https://user-images.githubusercontent.com/17882245/210377514-8b38ff90-7cbf-461f-b408-1f2b08ef7b62.png)

This means that we need to wrap the sentence with HTML:
```
div [ class "content" ]
        [ translateText language Translate.SendViaWhatsAppPhoneUpdateAtProfileQuestionPrefix
        , span [] [ text person.name ]
        , translateText language Translate.SendViaWhatsAppPhoneUpdateAtProfileQuestionSuffix
        , span [] [ text phoneNumber ]
        , text "?"
        ]
```

Since our Translate works with pure strings right now, we implement it this way.
When translation will be added, we'll review alternatives, if current structure is problematic.